dotCMS/src/main/java/com/dotmarketing/portlets/contentlet/business/HostAPIImpl.java

```diff
@@ -94,11 +93,13 @@ public Host findDefaultHost(User user, boolean respectFrontendRoles) throws DotS
         }
 
         try {
-            ContentType type =APILocator.getContentTypeAPI(APILocator.systemUser(), true).find("Host");
-
             List<Contentlet> list = null;
             try{
-                list = APILocator.getContentletAPI().search("+structureInode:" + type.inode() + " +working:true +host.isdefault:true ", 0, 0, null, APILocator.getUserAPI().getSystemUser(), respectFrontendRoles);
+                StringBuilder defaultHostQuery = new StringBuilder();
+                defaultHostQuery.append(String.format("+contentType:%s", HOST_VELOCITY_VAR_NAME));
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Define a constant instead of duplicating this literal "+contentType:%s" 6 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1192)